addons/api/addon/workers/web-worker.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { PWBWorker } from 'promise-worker-bi';
+import sqlite3InitModule from '@sqlite.org/sqlite-wasm';
+
+const promiseWorker = new PWBWorker();
+promiseWorker.register((message) => {
+  console.log(`Received from main: ${message}`);
+});
+
+const initializeSQLite = async () => {
```

Comment:
hmm so does every web worker need to initialize sql lite to query the db or is this just a temp implementation?